src/modules/caen/CaenInstrument.cpp

```diff
@@ -129,6 +129,8 @@ void CaenInstrument::dumpReadoutToFile(DataParser::CaenReadout &Data) {
 }
 
 void CaenInstrument::processReadouts() {
+  XTRACE(DATA, DEB, "Reference time is %u", ESSReadoutParser.Packet.Time
+          .TimeInNS);
```

Comment:
The default linebreaking makes this a bit less readable. At least the two last ones should be on same line 
- if needed move the comment to above the function.